openeo/util.py

```diff
@@ -627,3 +630,74 @@ def get(self, fraction: float) -> str:
         width = self.width - len(self.left) - len(self.right)
         bar = self.bar * int(round(width * clip(fraction, min=0, max=1)))
         return f"{self.left}{bar:{self.fill}<{width}s}{self.right}"
+
+
+class EPSGCodeNotFound(Exception):
```

Comment:
"not found" seems like a strange expressions for a CRS code, maybe make it "invalid EPSG code". Or just keep it to a vanilla `ValueError` with a good message?

Reply:
This was indeed not ideal, I was in fact already planning to remove that one.
Now these are ValueErrors. (And typerrors for anything that is not str, int or None)